TheraLang.Web/Controllers/ParticipationController.cs

```diff
@@ -2,32 +2,31 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.Identity;
-using Piranha.AspNetCore.Identity.Data;
 using System.Linq;
 using AutoMapper;
 using TheraLang.BLL.DataTransferObjects;
 using TheraLang.BLL.Interfaces;
 using TheraLang.Web.ViewModels;
 using Microsoft.AspNetCore.Authorization;
-using TheraLang.Web.Extensions;
 
 namespace TheraLang.Web.Controllers
 {
     [Route("api/participants")]
     [ApiController]
     public class ParticipationController : ControllerBase
     {
-        public ParticipationController(IProjectParticipationService projectParticipationServiceservice, IUserManagementService userManager, IProjectService projectService)
+        public ParticipationController(IProjectParticipationService projectParticipationServiceservice, IUserManagementService userManager, IProjectService projectService, IAuthenticateService authenticateService)
         {
             _projectParticipationServiceservice = projectParticipationServiceservice;
             _userManager = userManager;
-            _projectService = projectService;
```

Comment:
Field 'ParticipationController._projectService' is never assigned to, and will always have its default value null
Probably removed by a mistake